code/datums/spacelaw/major.dm

```diff
@@ -0,0 +1,65 @@
+datum/spacelaw/major/contraband
+	name = "Article 300 Possession of Contraband"
+	desc = "To be in the possession of contraband items. Being in the possession of two or more contraband items, or any restricted Syndicate gadget or weapon, makes you an Enemy of the Corporation (see above). In addition, being caught committing a Major Crime or higher with any of these items makes you an Enemy of the Corporation as well. Personnel turning in these items to Security of their own volition are exempt from this law."
```

Comment:
can you please `\` this only multiple lines, so we don't have to scroll so far over?